connector/connect/common/src/main/protobuf/spark/connect/relations.proto

```diff
@@ -601,3 +602,19 @@ message Unpivot {
   // (Required) Name of the value column.
   string value_column_name = 5;
 }
+
+message ToSchema {
+  // (Required) The input relation.
+  Relation input = 1;
+
+  // (Required) The user provided schema.
+  //
+  // The Sever side will update the dataframe with this schema.
+  oneof schema {
```

Comment:
not related to this PR, I am wondering whether we can unify the schema with String Literal (DDL or JSON)

Reply:
This requires clients side to implement more. For example, client side always convert StructType to string representation then Spark will convert it back.

This basically asks clients to understand Spark's protocol on the string based schema and then implement it right?